samples/snippets/csharp/System.Diagnostics/BooleanSwitch/Overview/remarks.cs

```diff
@@ -1,21 +1,21 @@
-﻿using System;
-using System.Diagnostics;
-
-public class SomeClass
-{
-// <Snippet2>

```

Comment:
Overview has two Main methods, so I guess compiling this code snippet will fail?

Reply:
Yes, it would fail. Adding the project files and fixing compilation errors will have to happen in a separate PR. There's a lot of manual effort involved in that.